.changeset/config.json

```diff
@@ -1,10 +1,14 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
-  "changelog": ["@changesets/changelog-github", { "repo": "ForgeRock/forgerock-javascript-sdk" }],
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "ForgeRock/forgerock-javascript-sdk" }
+  ],
   "fixed": [["@forgerock/*"]],
```

Comment:
This is important to note. This reads ALL package.json files `name` field. so if we prefix any package.json `name` as "@forgerock/" we will be trying to publish it.

We could do diligence and ensure we have `private` on all of the packages we do not want to publish but i think we should just try to avoid `@forgerock/` nameing for anything that doesn't exist in `packages/`.